### ISSUE TITLE
LeoFS 1.3.6 - MDC not replicating data

### ISSUE THREAD

**Issue by pannon** [leo-project/leofs#816]
In the latest 1.3.6 release, MDC data replication is not working at all.

Users (access/secret keys) and buckets are replicated correctly across DC's but uploading data to a bucket never leaves the origin DC.

Also `leofs-adm remove-cluster` results in the following error every time: `[ERROR] Could not connect`.

Is there any extra step required to configure MDC on the storage nodes since v1.3.2?

(We have a working MDC replication based on v1.3.2 release.)

**Comment by yosukehara** [comment]
@pannon Thank you for your report. 

I've recognized the multi-cluster-info table has not been synchronized between leo_manager and each leo_storage node. I'm going to fix it from now.

**Comment by pannon** [comment]
Thanks @yosukehara , happy to test if/when required.

**Comment by yosukehara** [comment]
@pannon I've found the root cause which is leo_storage's configuration issue as below and fixed it.

```diff
## https://github.com/leo-project/leofs/commit/3c5792daf4911bfee552cfa170066e770bf89d2f

diff --git a/apps/leo_storage/priv/leo_storage.schema b/apps/leo_storage/priv/leo_storage.schema
index 233653f..5a96756 100644
--- a/apps/leo_storage/priv/leo_storage.schema
+++ b/apps/leo_storage/priv/leo_storage.schema
@@ -1077,7 +1077,7 @@
  "leo_redundant_manager.sync_new_cluster_mod",
  [
   {datatype, atom},
-  {default, 'leo_storage_handle_sync'}
+  {default, 'leo_storage_handler_sync'}
  ]}.
```

You can easily fix it to replace `leo_storage/priv/leo_storage.schema` of your system with [leo_storage.schema](https://raw.githubusercontent.com/leo-project/leofs/develop/apps/leo_storage/priv/leo_storage.schema).

**Comment by pannon** [comment]
Nice one, the good old typo problem...
I am going to test this on our MDC setup today and report back.

**Comment by pannon** [comment]
Confirmed, after updating `leo_storage.schema` MDC replication works as expected.
@yosukehara thanks for resolving this issue.

**Comment by yosukehara** [comment]
@pannon Thanks for testing again. I've closed this issue.

**Comment by yosukehara** [comment]
@pannon We have decided that we release v1.3.7 in this week, which includes this fix.

**Comment by pannon** [comment]
There is also the `leofs-adm remove-cluster` problem, which results in `[ERROR] Could not connect`.

Is this an easy fix which could be included in v1.3.7?

**Comment by yosukehara** [comment]
@pannon Thanks for your report again. I've strictly checked the multi dc replication feature, then fixed some issues which includes the remote cluster's issue. We're going to seriously test the feature before releasing v1.3.7.

- PR: leo-project/leofs/pull/818

**Comment by yosukehara** [comment]
@pannon We tested the latest MDC replication implementation on our environment, then confirmed that we fixed MDC-replication related issues. If you have time, you can check [the latest master-branch](https://github.com/leo-project/leofs) _(but not tagged v1.3.7 yet)_.

**Comment by pannon** [comment]
Thanks @yosukehara. When do you expect to tag v1.3.7 officially?

**Comment by yosukehara** [comment]
@pannon 
>  When do you expect to tag v1.3.7 officially?

We're planning to release v1.3.7 tomorrow (Sep 12th) after finishing all of its tests.

**Comment by yosukehara** [comment]
@pannon If you confirmed this issue is fixed on v1.3.7, please close this issue.

**Comment by pannon** [comment]
Thanks @yosukehara
